person.rb

```diff
@@ -24,4 +31,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] This function should have a way to pass parameters to the Rental class, so obviously, we can't add rental to a rental. Here we should think about a date, a book, and a person getting a book `Rental.new(date, book, self)`. Your function should take three parameters so that they are passed to a Rental class.